### DISCUSSION TITLE
How to convert pred_masks to polygons?

### DISCUSSION THREAD

**Discussion by miladfa7** [facebookresearch/detectron2#3940] [General]
In my work, I need to convert the model output of polygon points instead of pred_masks.

In the annotated dataset, an object in an image, we have 15 segmentation points that we do not have in the model output(pred_masks). What can we do to have these segmentation points for objects?

**Comment by zhanghang1989** [comment]
If the dataset is generated by model using pseudolabel, you can simply set 
https://github.com/facebookresearch/detectron2/blob/e3053c14cfcce515de03d3093c0a7b4e734fa41f/detectron2/config/defaults.py#L87
to bitmask.
